test/e2e/notifier/create/create.go

```diff
@@ -55,7 +66,14 @@ func (c *context) testCreateResource(t *testing.T) {
 			err := json.Unmarshal([]byte(lastSeenMsg), &m)
 			assert.NoError(t, err, "message should decode properly")
 			assert.Equal(t, c.Config.Communications.Slack.Channel, m.Channel)
-			assert.Equal(t, test.Expected.Attachments, m.Attachments)
+			assert.Equal(t, test.ExpectedSlackMessage.Attachments, m.Attachments)
+
+			// Get last seen webhook payload
+			lastSeenPayload := c.GetLastReceivedPayload()
```

Comment:
This may fail if webhook is not called by the time this function call is made. Should be have retries here ?

Reply:
I guess the delay introduced for slack and time for the assertion of  slack notifier would be sufficient for the mock server receiving data. your thought.?
```
// Get last seen slack message
time.Sleep(time.Second)
```

Reply:
Ok. Looks good for now.